src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.bar {
+  position: relative;
+  margin-top: 20px;
+}
+
+.big {
+  padding-right: 62px;
+}
+
+.small {
+  padding-right: 33px;
+}
```

Comment:
Wait, why do you need this paddings? Your input should take all free width from page